src/Core/Users/login.class.php

```diff
@@ -0,0 +1,79 @@
+<?php
+require_once "UserInterface.php";
+
+Class login implements UserInterface{
+
+	public $_email;
+	public $_password;
+	public $_firstName;
+	public $_lastName;
+	public $_createdAt;
+	public $_updatedAt;
+	
+	public function __construct($email, $password){
+		$this->setEmail($email);
+		$this->setPassword($password);
+	}
+	
+	public function setEmail($email){
+		$this->_email = $email;
+	}
+	
+	public function getEmail(){
+		return $this->_email;
+	}
+	
+	public function setPassword($password){
+		$this->_password = $password;

```

Comment:
Please check the interface docs. The password should be hashed before assigning it to a variable.